src/command.c

```diff
@@ -57,14 +58,19 @@ static void itoa_rbin(uint16_t value, char *out) {
 
     for (int8_t i = 0; i < 16; i++) {
         v = (value >> i) & 1;
-        if (dont_ignore_zeros || v) {
-            out[index++] = '0' + v;
-            dont_ignore_zeros = 1;
-        }
+        out[index++] = '0' + v;
     }
-
-    if (!dont_ignore_zeros) out[index++] = '0';
     out[index] = '\0';
+
+    if (!dont_ignore_zeros) {
```

Comment:
you don't need this check as `dont_ignore_zeros` is never changed above.

Reply:
Removed dont_ignore_zeros entirely